### PR TITLE
Docs: Fixed animation timeout and lint warning

### PR DESCRIPTION
**Pull request by Baccega**
## Docs website
- Fixed animation speed in landing page
- Fixed same key error on footer

### CODE REVIEW

File: js-packages/docs-website/src/theme/Footer/index.js [hunk 1]
```diff
@@ -87,7 +87,10 @@ function Footer() {
                           }}
                         />
                       ) : (
-                        <li key={item.href || item.to} className="footer__item">
+                        <li
+                          key={`${item.href || item.to}-${key}`}
```
Comment (giulioz) [inline js-packages/docs-website/src/theme/Footer/index.js:91]: Nice catch

File: js-packages/docs-website/src/components/SearchesAnimation.js [hunk 2]
```diff
@@ -228,12 +228,12 @@ export function SearchesAnimation() {
       const query = randomArr(queries);
       setQuery(query);
 
-      await sleep(500);
+      await sleep(300);
```
Comment (giulioz) [inline js-packages/docs-website/src/components/SearchesAnimation.js:231]: Are you sure this is enough for not breaking the animation loop? I am worried a new query can arrive before the animation has finished

Comment (Baccega) [reply js-packages/docs-website/src/components/SearchesAnimation.js:231]: https://codesandbox.io/s/ok9-forked-6hx5u?file=/src/index.tsx

**Comment by giulioz** [comment]
Looks good to me.
Thanks for your contribution!

**Pull request merged**